.env.example

```diff
@@ -0,0 +1,3 @@
+EZEX_PROXIER_ADDRESS=127.0.0.1:8081
+EZEX_PROXIER_ENABLE_FASTHTTP=false
+EZEX_PROXIER_PROXY_RULES=/foo|https://httpbin.org/get,/bar|https://google.com
```

Comment:
In Rules we don't need to mention the HTTP Method? Like Get, or Post, etc?

Reply:
Not need